src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -80,10 +80,32 @@ Learn more about [routing alerts with integrations](/product/alerts/create-alert
 
 ### Issue Owners
 
-[Issue owners](/product/issues/issue-owners/) can receive notifications (emails only) when an alert is triggered. If an issue owner is not configured or not found, either the email notification won’t be sent or it’ll be sent to all project members, depending on the following setting in **[Project] > Settings > Issue Owners**.
+[Issue owners](/product/issues/issue-owners/) can receive notifications (emails only) when an alert is triggered.
+
+<Note>
+
+For early adopters, these notifications are received by email or Slack depending on the [notification settings](/product/alerts/notifications/notification-settings/) of the issue owner.
+
+</Note>
+
+If an issue owner is not configured or not found, either the notification won’t be sent or it’ll be sent to all project members, depending on the following setting in **[Project] > Settings > Issue Owners**.
 
 ![A toggle indicating if all users are issue owners or not.](issue_owners2.png)
 
+### Team Slack Notifications
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).
+
+</Note>
+
+You can configure Slack channels to receive alert notifications for teams. This can be done in Slack by using the `/sentry link team` command in the desired channel. To view a team's associated Slack channel in [sentry.io](https://sentry.io), navigate to **Settings > Teams > [Team] > Notifications**.
```

Comment:
Might be worth clarifying that this is one to one relationship. Maybe change the first sentence here to:

"A team can configure a Slack channel in which to receive alert notifications." or something like that